test/models/concerns/episode_media_test.rb

```diff
@@ -0,0 +1,276 @@
+require "test_helper"
+
+class EpisodeMediaTest < ActiveSupport::TestCase
+  let(:c1) { build_stubbed(:content, status: "complete", position: 1) }
+  let(:c2) { build_stubbed(:content, status: "complete", position: 2) }
+  let(:ep) { build(:episode, segment_count: 2, contents: [c1, c2]) }
+
+  describe "#complete_media_ready?" do
```

Comment:
Again, these were mostly ported from the `episode_test.rb`.  But I filled in the missing coverage, sorted, etc.